task5/src/main/java/com/clouway/adapter/http/HttpServletContextListener.java

```diff
@@ -15,12 +16,13 @@
   public void contextInitialized(ServletContextEvent servletContextEvent) {
     ServletContext servletContext = servletContextEvent.getServletContext();
     servletContext.addFilter("ConnectionFilter", new ConnectionFilter()).addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/*");
-    servletContext.addFilter("SecurityFilter", new SecurityFilter(new PersistentUserRepository(new PerRequestConnectionProvider()))).addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/welcomehomepage");
+    servletContext.addFilter("SecurityFilter", new SecurityFilter(new PersistentSessionRepository(new PerRequestConnectionProvider()))).addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/useraccount");
     servletContext.addServlet("login", new LoginPage()).addMapping("/login");
-    servletContext.addServlet("logincontroller", new LoginController(new PersistentUserRepository(new PerRequestConnectionProvider()))).addMapping("/logincontroller");
+    servletContext.addServlet("logincontroller", new LoginController(new PersistentUserRepository(new PerRequestConnectionProvider()), new PersistentSessionRepository(new PerRequestConnectionProvider()))).addMapping("/logincontroller");
     servletContext.addServlet("registercontroller", new RegisterController(new PersistentUserRepository(new PerRequestConnectionProvider()))).addMapping("/registercontroller");
     servletContext.addServlet("register", new RegisterPage()).addMapping("/register");
-    servletContext.addServlet("welcomehomepage", new UserAccount()).addMapping("/welcomehomepage");
+    servletContext.addServlet("logoutcontroller",new LogoutController(new PersistentSessionRepository(new PerRequestConnectionProvider()))).addMapping("/logoutcontroller");
+    servletContext.addServlet("useraccount", new UserAccount()).addMapping("/useraccount");
```

Comment:
Suggestion: you can reorganize all this lines so code looks more professional like:
option1:
- page1  
- ctrl1  
- filter1  
- page2  
- ctrl2  
- filter2  
...  
option2 by functionality   
- page 1  
- page 2   
- page 3  
- ctrl1  
- ctrl2  
- filter1  
- filter2    
...  

Personally I would use second